models/src/main/scala/coop/rchain/models/StringSyntax.scala

```diff
@@ -0,0 +1,16 @@
+package coop.rchain.models
+
+import com.google.protobuf.ByteString
+import coop.rchain.shared.Base16
+
+trait StringSyntax {
+  implicit final def modelsSyntaxString(s: String): StringOps =
+    new StringOps(s)
+
+}
+class StringOps(private val s: String) extends AnyVal {
+  def unsafeToByteString: ByteString      = ByteString.copyFrom(unsafeDecodeHex)
```

Comment:
This should be `unsafeHexToByteString`, just with prefix _unsafe_ like decode. Please add a line of separation between these two converters.